src/fluree/db/json_ld/api.cljc

```diff
@@ -122,6 +123,34 @@
    (promise-wrap
      (jld-ledger/load conn address))))
 
+(defn alias->address
+  "Returns a core.async channel with the connection-specific address of the
+  given ledger-alias."
+  [conn ledger-alias]
+  (conn-proto/-address conn ledger-alias nil))
+
+(defn load
+  "Loads an existing ledger by its alias (which will be converted to a
+  connection-specific address first)."
+  [conn ledger-alias]
+  (promise-wrap
+    (go
+      (let [address (<! (alias->address conn ledger-alias))]
+        (log/debug "Loading ledger from" address)
+        (<! (jld-ledger/load conn address))))))
+
+(defn load-if-exists
+  "Attempts to load ledger-alias, returning a promise containing nil if it
+  doesn't exist, the ledger otherwise."
+  [conn ledger-alias]
+  (promise-wrap
+    (go
+      (let [address (<! (alias->address conn ledger-alias))
+            _ (log/debug "Attempting to load ledger from" address)
+            attempt (<! (jld-ledger/load conn address))]
+        (when-not (util/exception? attempt)
```

Comment:
Minor nitpick, but there could be an exception here from a different error other than the ledger not existing. In that case the issue would be swallowed and the user would assume the ledger doesn't exist when it does but it just has another problem. 

I don't think it's a big deal for now, but maybe we could look for a specific exception to ignore, and pass everything else through.

Reply:
True, true. I'll investigate. And/or eliminate this fn entirely. :)